esp/templates/program/modules/creditcardmodule_stripe/failure.html

```diff
@@ -44,11 +42,11 @@ <h3>Failure</h3>
 {% endif %}
 
 {% if error_type == "invalid" %}
-<b>Invalid request.</b>  This is usually caused by a duplicate request (e.g. "refreshing" your Web browser after paying).  It is possible that your payment went through successfully; please return to the payment page to check.
+<b>Invalid request.</b>  This is usually caused by a duplicate request (such as "refreshing" your web browser after paying).  It is possible that your payment went through successfully; <b>please return to the payment page to check</b>.
```

Comment:
in 2018 we can probably remove the quotes from "refreshing"

Reply:
+1 it's kinda fun, kinda weird to see what's changed in ~10 years